<launch>

<!-- include file="$(find realsense_camera)/launch/sr300_nodelet_rgbd.launch)" -->

<!-- <node pkg="ras_project_camera" name="camera_pcl" type="detection" output="screen"> 
<param name="min_object_size" type="int" value="30" />
</node> -->

<node name="color_detection" pkg="ras_project_camera" type="color_detection" output="screen">
<param name="green_h_max" type="int" value="75"/>
<param name="green_s_max" type="int" value="255"/>
<param name="green_v_max" type="int" value="255"/>
<param name="green_h_min" type="int" value="35"/>
<param name="green_s_min" type="int" value="80"/>
<param name="green_v_min" type="int" value="50"/>

<param name="blue_h_max" type="int" value="34"/>
<param name="blue_s_max" type="int" value="255"/>
<param name="blue_v_max" type="int" value="255"/>
<param name="blue_h_min" type="int" value="15"/>
<param name="blue_s_min" type="int" value="100"/>
<param name="blue_v_min" type="int" value="30"/>

<param name="yellow_h_max" type="int" value="110"/>
<param name="yellow_s_max" type="int" value="255"/>
<param name="yellow_v_max" type="int" value="255"/>
<param name="yellow_h_min" type="int" value="80"/>
<param name="yellow_s_min" type="int" value="40"/>
<param name="yellow_v_min" type="int" value="40"/>

<param name="morph" type="int" value="10"/>
<param name="nb_of_colors" type="int" value="2"/>
<param name="minTargetRadius" type="int" value="50"/>
<param name="maxTargetRadius" type="int" value="200"/>
</node>

<!-- <node name="camera_pcl" pkg="ras_project_camera" type="camera_pcl" output="screen"> </node> -->

</launch>
